### ISSUE TITLE
Example folder

### ISSUE THREAD

**Issue by cavill** [CulturalMe/meteor-slingshot#40]
It's no doubt my own incompetence at fault, but I'm unclear on how to implement the server-side part of Slingshot.

I wonder if there's an sample Meteor project anywhere that would show how to get set up (in the context of an entire project, as opposed to just the code samples in your README)?

Thanks.

**Comment by gsuess** [comment]
Yes we were considering that. However priority now is to extend the feature set. A pull request as well as suggestions on how the example app could look like are welcome.
